src/connection.ts

```diff
@@ -31,7 +33,10 @@ export async function createPeerToOffer(onSuccess: (offer: Signal) => void, onCh
     // Create the local connection and its event listeners
     const conn = new RTCPeerConnection({ iceServers: ICE_SERVERS })
 
-    const peer: Peer = { id: uuidV4(), conn }
+    const peer: Peer = { id: uuidV4(), username: 'not yet set', conn }
+    //peer.username = sessionStorage.getItem('username') || 'Anonymous'
+    console.log(peer.username + ' should be ' + sessionStorage.getItem('username'))
+    console.log('Create Peer to Offer Function')
```

Comment:
i think these were leftover from debugging, they should be safe to remove as well